.github/workflows/ci.yml

```diff
@@ -256,24 +256,36 @@ jobs:
         run: |
           set -euxo pipefail
           mkdir -p test/.tmp/debian-kernels/arm64
+
           # NB: a 4.19 kernel image for arm64 was not available.
-          # TODO: enable tests on kernels before 6.0.
-          # linux-image-5.10.0-23-cloud-arm64-unsigned_5.10.179-3_arm64.deb \
-          printf '%s\0' \
-            linux-image-6.1.0-16-cloud-arm64-unsigned_6.1.67-1_arm64.deb \
-          | xargs -0 -t -P0 -I {} wget -nd -nv -P test/.tmp/debian-kernels/arm64 ftp://ftp.us.debian.org/debian/pool/main/l/linux/{}
+          kernels=(
+            "linux-image-5.10.0-32-cloud-arm64-unsigned_5.10.223-1_arm64.deb"
+            "linux-image-6.1.0-16-cloud-arm64-unsigned_6.1.67-1_arm64.deb"
+            "linux-image-6.10.4-cloud-arm64-unsigned_6.10.4-1_arm64.deb"
+          )
+
+          for kernel in "${kernels[@]}"; do
+            echo "Downloading $kernel"
+            wget -nd -nv -P test/.tmp/debian-kernels/amd64 "ftp://ftp.us.debian.org/debian/pool/main/l/linux/$kernel"
+          done
 
       - name: Download debian kernels
         if: runner.arch == 'X64'
         run: |
           set -euxo pipefail
           mkdir -p test/.tmp/debian-kernels/amd64
-          # TODO: enable tests on kernels before 6.0.
```

Comment:
Ack

Reply:
Can you undo all the unrelated changes please?

Reply:
Done

Reply:
I meant the loops and arrays - there's no need for this, you can just add lines in the existing formulation

```
          printf '%s\0' \
            linux-image-6.1.0-16-cloud-arm64-unsigned_6.1.67-1_arm64.deb \
            linux-image-6.10.4-cloud-arm64-unsigned_6.10.4-1_arm64.deb \
```
this would work just fine.